src/services/meta/note_meta.ts

```diff
@@ -1,23 +1,25 @@
+import AttachmentMeta = require("./attachment_meta");
 import AttributeMeta = require("./attribute_meta");
 
 interface NoteMeta {
-    noteId: string;
-    notePath: string;
-    isClone: boolean;
-    title: string;
-    notePosition: number;
-    prefix: string;
-    isExpanded: boolean;
-    type: string;
-    mime: string;
+    noteId?: string;
+    notePath?: string[];
```

Comment:
Why was this changed to an array?

Reply:
@alexpietsch , I believe the original data type (from my previous PR) was incorrect. Note paths are expressed as an array of strings.

Reply:
I tested it, it is actually an array.